.github/workflows/main.yml

```diff
@@ -16,9 +16,9 @@ jobs:
       PRIVATE_REGISTRY_URL: ${{ secrets.PRIVATE_REGISTRY_URL }}
       TAG_DEVELOP: develop
       TAG_LATEST: latest
-      JDK_VERSION: 16
-      JDK_FILE: openjdk-16.0.1_linux-x64_bin.tar.gz
-      JDK_URL: https://download.java.net/java/GA/jdk16.0.1/7147401fd7354114ac51ef3e1328291f/9/GPL/openjdk-16.0.1_linux-x64_bin.tar.gz
+      JDK_VERSION: 17
+      JDK_FILE: openjdk-17_linux-x64_bin.tar.gz
+      JDK_URL: https://download.java.net/java/GA/jdk17/0d483333a00540d886896bac774ff48b/35/GPL/openjdk-17_linux-x64_bin.tar.gz
```

Comment:
Are there benefits for this approach over the configuration of the `setup java` action? The adoptium.net version should be a pretty good alternative, or would we miss out on anything?

Reply:
I just kept this the same way as it was before (Oracle OpenJDK) but we can switch to other... However, adoptium does not have version 17 yet. But should be soon: https://github.com/adoptium/adoptium/issues/74

I recall that there were some issues with Zuulu Java (tests failing randomly) when we were setting up GitHub Actions.

Reply:
Alright, let's keep it as is, and we can reconsider when it becomes available.

Reply:
I agree, but it is good that you raised this discussion as I forgot to update Dockefiles 😄 

When we switch to other JDK distribution, we will need to adjust the Dockefiles accordingly (and also suggest that distribution for development).